### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

**Comment by nabobalis** [comment]
We need to disable circle. Otherwise it will replace it like it did for my commit.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,11 @@ The SunPy 1.0 paper repository.
 Published Version
 =================
 
-https://circleci.com/gh/sunpy/sunpy-1.0-paper/441/artifacts/0//root/project/main.pdf
+The paper was published in ApJ on 2020 February 12. The published version can be found [here](https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a).
```
Comment (nabobalis) [inline README.md]:
```suggestion
The paper was published in ApJ on 2020 February 12 and can be found [here](https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a).
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ The SunPy 1.0 paper repository.
...
+
+The citation is
```
Comment (nabobalis) [inline README.md]: Do we want to have the bibtex too?

**Comment by nabobalis** [comment]
Stuart has now disabled circleci.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,11 @@ The SunPy 1.0 paper repository.
 Published Version
 =================
 
-https://circleci.com/gh/sunpy/sunpy-1.0-paper/441/artifacts/0//root/project/main.pdf
+The paper was published in ApJ on 2020 February 12 and can be found [here](https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a).
+
+The citation is 
+“The SunPy Project: Open Source Development and Status of the Version 1.0 Core Package,” The SunPy Community et al 2020 ApJ 890 68. doi:10.3847/1538-4357/ab4f7a
```
Comment (nabobalis) [inline README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -4,7 +4,11 @@ The SunPy 1.0 paper repository.
 Published Version
 =================
 
-https://circleci.com/gh/sunpy/sunpy-1.0-paper/441/artifacts/0//root/project/main.pdf
+The paper was published in ApJ on 2020 February 12 and can be found [here](https://iopscience.iop.org/article/10.3847/1538-4357/ab4f7a).
+
+The citation is
```
Comment (nabobalis) [inline README.md]:
```suggestion
How to cite this is found [here.](https://github.com/sunpy/sunpy/blob/master/sunpy/CITATION.rst)
```

**Review by wtbarnes — APPROVED** [review]

**Pull request merged**